homeassistant/components/statistics/sensor.py

```diff
@@ -764,7 +764,7 @@ def _stat_sum_differences(self) -> StateType:
         if len(self.states) >= 2:
             diff_sum = sum(
                 abs(j - i)
-                for i, j in zip(list(self.states), list(self.states)[1:], strict=True)
+                for i, j in zip(list(self.states), list(self.states)[1:], strict=False)
```

Comment:
Is this a bug?

Reply:
In this case both lists are deliberately of different length (because of removing the first element on the second argument)

Another solution would be this to keep both the same length:
```suggestion
                for i, j in zip(list(self.states)[:-1], list(self.states)[1:], strict=True)
```

Reply:
Probably best to set it to False for now and that could be done in a followup when its not part of a larger PR